CHANGES.md

```diff
@@ -1,3 +1,22 @@
+
+## Version 1.0.0-RC6 (January 23, 2021)
```

Comment:
Seems like this one should be RC7 and `Version 1.0.0-RC7 (April 18, 2020)` below should be removed

Reply:
Corrected, sorry I only intended to add the entry for RC7 that was missing.